docs/source/exporters/overview.mdx

```diff
@@ -1,3 +1,3 @@
 # Overview
 
-🤗 Optimum enables exporting models from PyTorch or TensorFlow to different formats through its `exporters` module. For now, only one exporting format is supported: ONNX.
+🤗 Optimum enables exporting models from PyTorch or TensorFlow to different formats through its `exporters` module. For now, only two exporting format are supported: ONNX and TFLite.
```

Comment:
Either TFLite or TensorFlow Lite

Reply:
?

Reply:
We can use either TensorFlow Lite as a naming or TFLite, maybe let's keep TFLite